deps.edn

```diff
@@ -46,4 +48,8 @@
                        :extra-deps  {thheller/shadow-cljs                {:mvn/version "2.11.8"}
                                      com.fulcrologic/semantic-ui-wrapper {:mvn/version "2.0.1"}
                                      binaryage/devtools                  {:mvn/version "1.0.0"}
-                                     org.clojure/tools.namespace         {:mvn/version "1.0.0"}}}}}
+                                     org.clojure/tools.namespace         {:mvn/version "1.0.0"}
+                                     cider/cider-nrepl                   {:mvn/version "0.25.6"}
+                                     iced-nrepl/iced-nrepl               {:mvn/version "1.1.1"}
```

Comment:
I'd prefer these stay part of your personal .clojure/deps.edn file. I do not like added dependencies in projects that I maintain that involve things I do not use because it shifts a maintenance burden to me that I do not want. I'm ok with the condo configs.